src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -1923,8 +1923,22 @@ void CodeGen::codeGenInstruction(ByteCodeOp operation)
             llvm::Value* value = operandStack.pop_back();
             operandStack.push_back(builder.CreateTrunc(value, builder.getInt32Ty()));
         },
-        // TODO: LAnd
-        // TODO: LCmp
+        [&](LAnd)
```

Comment:
I beleive you should be able to merge this and `LOr` togther with `IAnd` & friends using just `OneOf<LAnd, IAnd>` and so on.
The implementations are identical as far as I can tell.